From d1dd7f7703b4f40d2fbb830e28969b31b8a1673e Mon Sep 17 00:00:00 2001
From: Sumit Bose <sbose@redhat.com>
Date: Tue, 3 Jul 2018 11:30:07 +0200
Subject: [PATCH 22/83] sysdb_ldb_msg_attr_to_certmap_info: set
 SSS_CERTMAP_MIN_PRIO

Make sure that priority is always set.

Related to https://pagure.io/SSSD/sssd/issue/3500

Reviewed-by: Jakub Hrozek <jhrozek@redhat.com>
---
 src/db/sysdb_certmap.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/db/sysdb_certmap.c b/src/db/sysdb_certmap.c
index e61cc05..0bb7ebc 100644
--- a/src/db/sysdb_certmap.c
+++ b/src/db/sysdb_certmap.c
@@ -22,6 +22,7 @@
 
 #include "util/util.h"
 #include "db/sysdb_private.h"
+#include "lib/certmap/sss_certmap.h"
 
 static errno_t sysdb_create_certmap_container(struct sysdb_ctx *sysdb,
                                               bool user_name_hint)
@@ -327,6 +328,8 @@ errno_t sysdb_ldb_msg_attr_to_certmap_info(TALLOC_CTX *mem_ctx,
         }
 
         map->priority = (uint32_t) tmp_uint;
+    } else {
+        map->priority = SSS_CERTMAP_MIN_PRIO;
     }
 
     tmp_el = ldb_msg_find_element(msg, SYSDB_CERTMAP_DOMAINS);
-- 
2.9.5

